src/components/SideBar.astro

```diff
@@ -34,7 +34,7 @@ import { Image } from "@astrojs/image/components";
       <li><a id="store" href="/store">Store</a></li>
       <li><a id="blog" href="/blog/">Blog</a></li>
       <li><a id="cv" href="/cv">CV</a></li>
-      <li><a href="mailto:contact.manuelernestog@gmail.com">Contact</a></li>
+      <li><a href="mailto:contact.manuelernestog@gmail.com" target="_blank" referrerpolicy="no-referrer-when-downgrade">Contact</a></li>
```

Comment:
Hi @CarlosZBent Arigato for the PR :)

🤔 The mailto: supposes to open the email client in the computer/phone of the user. 

Why should we add a target blank ?
And why use no-referrer-when-downgrade ?

Reply:
Hey @manuelernestog . That indeed opens the email client, but, for example, I have no desktop email client, I use gmail by default. So it opens it in the same tab as the page instead of sending me to a new one. I imagine there must be a lot of users who similar to me, have gmail, outlook web, or similars as default email clients. So in that case, having `target=_blank` doesn't matter to the mobile or desktop client users, but it improves the browser email client experience.

The `no-referrer-when-downgrade` is just a good middle ground for the referrer_policy. It's not showing all the data to everyone, but it's also not hiding it from everyone. It only shows the origin headers if the destination protocol security level stays the same or improves. [Link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy).

Reply:
Excellent !!